### ISSUE TITLE
show amount of agents graphically

### ISSUE THREAD

**Issue by bartrail** [phreeza/cells#17]
as an addon to the UI, maybe display a graph that ticks along to see the increasing/decreasing of a party..

**Comment by phreeza** [comment]
Good idea, I would suggest a log scale for this.

**Comment by bartrail** [comment]
whats a log scale? :)

**Comment by phreeza** [comment]
A logarithmic scale, so plotting log(N) instead of N. Good for graphing things that span several orders of magnitude, as we have here. In practice this would mean the height difference between 1 and 10 cells is the same as between 10 and 100, for example.

**Comment by bartrail** [comment]
ah haven't thought about 'logarithmic' ;) 
maybe we add an extra "toolbar" area to the top/side where a graph or some more informations can be displayed

**Comment by phreeza** [comment]
yes, that would be nice. team name, unit count/graph, current/remaining tics come to mind as content for this.
Are you on it?

**Comment by bartrail** [comment]
sounds great. :) I think vertical at the left with a with about 100px makes more sense than at the top.. 
I can see what I can do :) calling for help if necessary

**Comment by exupero** [comment]
Perhaps this should be a pop-up the user can open with a keyboard shortcut?
